easybuild/easyconfigs/p/pytest/pytest-4.6.11-foss-2023a-Python-2.7.18.eb

```diff
@@ -0,0 +1,100 @@
+easyblock = 'PythonBundle'
+
+name = 'pytest'
+version = '4.6.11'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://pytest.org'
+description = """pytest: simple powerful testing with Python"""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
```

Comment:
This should be `GCC`, no need for a full toolchain that also includes MPI + BLAS/LAPACK for `pytest`...